Adding note as suggested. (#4011)
MicrosoftDocs/windows-itpro-docs: windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md

Though much Windows BitLocker [documentation](bitlocker-overview.md) has been published, customers frequently ask for recommendations and pointers to specific, task-oriented documentation that is both easy to digest and focused on how to deploy and manage BitLocker. This article links to relevant documentation, products, and services to help answer this and other related frequently-asked questions, and also provides BitLocker recommendations for different types of computers.

>[!IMPORTANT]

> > Microsoft BitLocker Administration and Monitoring (MBAM) capabilities would be offered from [SCCM in on-prem scenarios](https://docs.microsoft.com/microsoft-desktop-optimization-pack/mbam-v25/viewing-mbam-25-reports-for-the-configuration-manager-integration-topology) in a future.

Reviewer: ```suggestion
> Microsoft BitLocker Administration and Monitoring (MBAM) capabilities will be offered from [SCCM in on-prem scenarios](https://docs.microsoft.com/microsoft-desktop-optimization-pack/mbam-v25/viewing-mbam-25-reports-for-the-configuration-manager-integration-topology) in the future.
```

<br />

<a id="powershell"></a>

> # **Powershell**

Reviewer: ```suggestion
# **PowerShell**
```
